.github/workflows/R-CMD-check.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
     env:
       GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
       R_KEEP_PKG_SOURCE: yes
+      R_NOT_CRAN: yes
```

Comment:
Is there any reason not to use the widely used `NOT_CRAN="true"`?

Reply:
no good reason, no

Reply:
I recommend using `NOT_CRAN`.

For example, the r-lib's action has already set it as shown below.
https://github.com/grantmcdermott/tinyplot/actions/runs/8009321355/job/21877708151?pr=122#step:6:16

Reply:
At GitHub you can simple check for `CI` or `GITHUB_ACTION` being present.  Actually checking for something that has to be present for an action to take place always strikes me as cleaner.  But that may just be me.

Reply:
Okay, let's go for `NOT_CRAN` per @eitsupi's comment, since that seems to be a norm.